scrum.py

```diff
@@ -55,32 +58,63 @@ def is_max_number_of_user_story_per_sprint_valid(self, sprint_model):
+            if(isTrelloEnabled):
+                created_sprint_on_trello = self.create_new_sprint_on_trello(sprint_model, config)
+                self.create_sprint_user_stories_trello(sprint_model.userStories, created_sprint_on_trello, config)
+            if(isJiraEnabled):
+                self.create_sprint_user_stories_jira(sprint_model.userStories, config)
 
 
-    def create_sprint_user_stories(self, sprint_user_stories_model, created_sprint):
+    def create_sprint_user_stories_trello(self, sprint_user_stories_model, created_sprint, config):   
         # TODO: Move this get call to one logical higher level (call it once, not for each sprint!)
-        all_board_members = self.get_all_board_members()
-        all_board_labels = self.get_all_board_labels()
+        all_board_members = self.get_all_board_members(config)
+        all_board_labels = self.get_all_board_labels(config)
+      
         for user_story_model in sprint_user_stories_model: 
             story_member_ids = self.get_story_member_ids(user_story_model, all_board_members)
             story_label_ids = self.get_story_label_ids(user_story_model, all_board_labels)
-            story_payload = {
+            story_payload_trello = {
                     'idList': created_sprint['id'],
-                    'key': "9519ec4ca00591297f8bb4e7e184a841",
-                    'token': "013c3b97e0290d108573fb6d150a8bf32982b84150c20a4d372bf701dabe8d82",
+                    'key': config["apiSecurity"]["trelloKey"],
+                    'token': config["apiSecurity"]["trelloToken"],
                     'name': '(' + str(user_story_model.userStoryDetails.storyPoints) + ') ' + user_story_model.name,
                     'desc': user_story_model.userStoryBody.storyDescription.value,
                     'idMembers': story_member_ids,
                     'idLabels': story_label_ids
                 }
-                
-            self.create_new_ticket_on_trello(story_payload)
+            self.create_new_ticket_on_trello(story_payload_trello)
+            
 
+    def create_sprint_user_stories_jira(self, sprint_user_stories_model, config):
+    
+        for user_story_model in sprint_user_stories_model: 
+            story_payload_jira = {
+                "fields":{
+                    "project":{
+                        "key":config["boardsInfo"]["jiraProjectKey"],
+                    },
+                    "summary":'(' + str(user_story_model.userStoryDetails.storyPoints) + ') ' + user_story_model.name,
+                    "description": user_story_model.userStoryBody.storyDescription.value,
+                    "issuetype":{
+                        "name":"Story"
+                    },
+                    "assignee": {
+                         "id": '627c209023d61e006fc50f11'
+                     },
+                    "labels": [
+                        "bugfix",
+                        "blitz_test"
+                     ],      
```

Comment:
Again, why we have hardcoded these values for the labels?

Reply:
fixed, now we have dynamic labels from Grammarly